src/app/components/Snapshots/SnapshotCard.tsx

```diff
@@ -39,12 +40,13 @@ export const SnapshotCard: FC<SnapshotCardProps> = ({
   title,
   label,
   withContentPadding = true,
+  withConstantHeight = false,
 }) => {
   return (
     <StyledCard>
       <CardHeader component="h5" title={title} sx={{ pb: 0, pl: 4, pt: 4 }} />
       <StyledCardContent withContentPadding={withContentPadding}>{children}</StyledCardContent>
-      {(badge || label) && (
+      {(badge || label || withConstantHeight) && (
```

Comment:
This solves this issue: 
![image](https://github.com/oasisprotocol/explorer/assets/2093792/024fbcc5-8d36-4d0e-bfce-cae1816dab37)